components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -1339,6 +1414,18 @@
                                 </div>
                             </td>
                         </tr>
+                        <!-----jwks text box JWKS TEXT BOX------->
```

Comment:
HTML comments on need `<!--` in the beginning and `-->` in the end.

Reply:
corrected